src/styles/base.scss

```diff
@@ -0,0 +1,8 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
```

Comment:
Move font family to the html selector

Reply:
done

Reply:
<img width="623" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/0138c7e8-afe7-40eb-a079-4b1e3b6dd712">
still in the * selector